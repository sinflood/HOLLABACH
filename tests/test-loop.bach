comp Banjo{
loop 4{
[C3/2]
}
}
