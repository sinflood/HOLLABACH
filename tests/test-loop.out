Timing Resolution (pulses per quarter note)
4

Instrument,105,,

Tick,Note(0-127),Velocity(0-127),
0,48,2
16,48,2
32,48,2
48,48,2
